### ISSUE TITLE
Cleanup core examples once macros can inspect source file

### ISSUE THREAD

**Issue by AaronO** [denoland/deno#13939]
Following https://github.com/denoland/deno/pull/13861, some of `core/examples/*` have a "hackish" `use deno_core::*;` statement to workaround the fact that the `#[op]` macro can't detect that they should use `deno_core::` instead of `crate::`.

This is blocked on https://github.com/rust-lang/rust/issues/54725

**Comment by bartlomieju** [comment]
It appears this issue is no longer relevant since we can just do `use deno_core::op`.
